packages/firestore/android/src/main/java/io/capawesome/capacitorjs/plugins/firebase/firestore/classes/options/WriteBatchOperation.java

```diff
@@ -11,11 +11,19 @@ public class WriteBatchOperation {
     private String type;
     private String reference;
     private Map<String, Object> data;
+    private SetOptions options;
```

Comment:
```suggestion
    @Nullable
    private SetOptions options;
```

Please add this annotation to make it clear that this value can be null.